pyproject.toml

```diff
@@ -12,46 +12,43 @@ dependencies = [
     "python-magic~=0.4.27",
     "python-dotenv==1.0.0",
     # Adapter changes
-    "llama-index==0.10.38",
-    # Temporary hack to get out of llama-index issues
-    # To be removed once llama-index version is upgraded
-    "llama-index-core==0.10.56",
+    "llama-index==0.10.58",
     "tiktoken~=0.4.0",
     "transformers==4.37.0",
     "llama-index-embeddings-google==0.1.5",
```

Comment:
Staying on 0.1.5 for llama-index-embedding-google here even though latest is 0.1.5 in order to resolve lock conflicts